### PR TITLE
Add JS/CSS checking using Prettier (#274)

### PR DESCRIPTION
**Pull request by mcantelon**
Added Javascript and CSS file testing, using the Prettier tool, and updated formatting in JS/CSS files to conform to Prettier's standard.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -21,3 +21,8 @@ repos:
   hooks:
   - id: markdownlint
     args: ["--config", ".markdownlint.jsonc"]
+- repo: https://github.com/pre-commit/mirrors-prettier
+  rev: ''  # Use the sha / tag you want to point at
```
Comment (sevein) [inline .pre-commit-config.yaml:25]: What's the meaning of the emptry string?

**Review by sevein — APPROVED** [review]
:+1:

File: .pre-commit-config.yaml (hunk 1 above)
Comment (mcantelon) [reply .pre-commit-config.yaml]: Oops... fixed! Thanks!

**Comment by mcantelon** [comment]
Thanks @sevein !!!

**Pull request merged**